l10n_ar_sale/models/sale_order.py

```diff
@@ -63,13 +63,19 @@ def _get_sale_checkbook(self):
     @api.model_create_multi
     def create(self, vals):
         for val in vals:
-            if self.env.user.has_group('l10n_ar_sale.use_sale_checkbook') and \
-                val.get('name', _('New')) == _('New') and \
-                    val.get('sale_checkbook_id'):
-                sale_checkbook = self.env['sale.checkbook'].browse(
-                    val.get('sale_checkbook_id'))
-                val['name'] = sale_checkbook.sequence_id and\
-                    sale_checkbook.sequence_id._next() or _('New')
+            sale_checkbook_id = val.get('sale_checkbook_id')
+            if self.env.user.has_group('l10n_ar_sale.use_sale_checkbook') and sale_checkbook_id:
+                if val.get('name', _('New')) == _('New'):
```

Comment:
```suggestion
            if self.env.user.has_group('l10n_ar_sale.use_sale_checkbook') and 
                  val.get('name', _('New')) == _('New') and \
                  sale_checkbook_id:
```
por que no juntar ambos if? de los contrario tenemos más cambios en el diff